docs/pxc-pro.md

```diff
@@ -8,6 +8,7 @@ The following capabilities have been tested for {{release}} and are available in
 
 | Name                                | Available since | Description  | 
 | ----------------------------------- | ------------- | -------------|
+|[Reduced backup lock time](https://docs.percona.com/percona-xtrabackup/8.4/reduction-in-locks.html)|8.4.6-6|The Percona XtraDB Cluster Pro `xtrabackup` SST (State Snapshot Transfer) method, based on XtraBackup Pro, now uses the [Reduced backup lock time](https://docs.percona.com/percona-xtrabackup/8.4/reduction-in-locks.html) feature. This enhancement minimizes blocking on the **Donor node** during SST process while the backup is being prepared. The reduced lock feature is enabled by default with the `--lock-ddl=REDUCED` option. For more details about the `--lock-ddl` option and its available values, refer to the [xtrabackup command-line options](https://docs.percona.com/percona-xtrabackup/8.4/xtrabackup-option-reference.html#lock-ddl) documentation.|
```

Comment:
> The reduced lock feature is enabled by default with the `--lock-ddl=REDUCED` option. For more details about the `--lock-ddl` option and its available values, refer to the

Maybe this way:
Usage of PXB's reduced lock feature is enabled by default. It can be changed by specifying the desired lock_ddl value in [xtrabackup] section of my.cnf. For more details about the `lock_ddl` option and its available values, refer to the  ...